client/components/videos-ui/index.jsx

```diff
@@ -127,6 +127,7 @@ const VideosUi = ( { shouldDisplayTopLinks = false, onBackClick = () => {} } ) =
 							videoRef={ videoRef }
 							videoUrl={ currentVideo.url }
 							isPlaying={ isPlaying }
+							poster={ currentVideo.poster ? currentVideo.poster : false }
```

Comment:
Slightly pedantic perhaps but I feel like "null" makes more sense as a fallback value here, instead of "false".